public/app/plugins/datasource/influxdb/components/editor/config-v2/AdvancedDBConnectionSettings.tsx

```diff
@@ -140,6 +127,18 @@ export const AdvancedDbConnectionSettings = (props: Props) => {
               />
             </InlineField>
           </InlineFieldRow>
+          {options.jsonData.version === InfluxVersion.SQL && (
```

Comment:
This needs a unit test 🙏

Reply:
This already has a test: https://github.com/grafana/grafana/blob/238961d3ea5dbc66e149c619b757d234b8b27cf6/public/app/plugins/datasource/influxdb/components/editor/config-v2/AdvancedDBConnectionSettings.test.tsx#L48

Reply:
This has a test 🤔 

https://github.com/grafana/grafana/blob/238961d3ea5dbc66e149c619b757d234b8b27cf6/public/app/plugins/datasource/influxdb/components/editor/config-v2/AdvancedDBConnectionSettings.test.tsx#L48